pom.xml

```diff
@@ -381,6 +381,11 @@ under the License.
       <version>3.19.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.5</version>
```

Comment:
Should be 1.7.32 by now or make it a properly for API and impl

Reply:
yup sure. I was just aligning with api version.